client/hotrod-client/src/test/java/org/infinispan/client/hotrod/SslTest.java

```diff
@@ -170,8 +173,8 @@ public void testClientAuth() throws Exception {
    }
 
    // Note: with Netty this started to throw SSLException instead of SSLHandshakeException
-   public void testClientAuthWithAnonClient() throws Exception {
-      Exceptions.expectExceptionNonStrict(TransportException.class, IOException.class, () -> initServerAndClient(true, true, true, false, false, false));
+   public void testClientAuthWithAnonClient() {
+      Exceptions.expectExceptionNonStrict(TransportException.class, DecoderException.class, () -> initServerAndClient(true, true, true, false, false, false));
```

Comment:
This is failing, as a `ClosedChannelException` is being thrown instead of the `DecoderException`.